docs/src/components/productX/XComponents.tsx

```diff
@@ -1,139 +1,47 @@
-import ShowChartRounded from '@mui/icons-material/ShowChartRounded';
-import BarChartRounded from '@mui/icons-material/BarChartRounded';
 import Section from 'docs/src/layouts/Section';
 import SectionHeadline from 'docs/src/components/typography/SectionHeadline';
 import GradientText from 'docs/src/components/typography/GradientText';
-import Item, { Group } from 'docs/src/components/action/Item';
-import Highlighter from 'docs/src/components/action/Highlighter';
+import XComponentsSwitcher from 'docs/src/components/productX/XComponentsSwitcher';
 import XGridFullDemo from 'docs/src/components/productX/XGridFullDemo';
 import XDateRangeDemo from 'docs/src/components/productX/XDateRangeDemo';
 import XTreeViewDemo from 'docs/src/components/productX/XTreeViewDemo';
 import XChartsDemo from 'docs/src/components/productX/XChartsDemo';
-import XSparklineDemo from 'docs/src/components/productX/XSparklineDemo';
-import More from 'docs/src/components/action/More';
-import ROUTES from 'docs/src/route';
-import IconImage from 'docs/src/components/icon/IconImage';
-
-const DEMOS = ['Data Grid', 'Date and Time Pickers', 'Charts', 'Tree View', 'Sparkline'];
-const WIP = DEMOS.slice(4);
-
-function PrefetchImages() {
-  function makeImg(component: 'sparkline' | 'chart', mode: string, num: number) {
-    return {
-      loading: 'lazy' as const,
-      width: 100,
-      src: `/static/branding/mui-x/${component}-${mode}${num}.png`,
-    };
-  }
-  return (
-    <Box
-      sx={{
-        width: 0,
-        height: 0,
-        visibility: 'hidden',
-        '& > img': {
-          position: 'absolute',
-        },
-      }}
-    >
-      {[...Array(2)].map((_, index) => (
-        <React.Fragment key={index}>
-          <img alt="" {...makeImg('sparkline', 'light', index + 1)} />
-          <img alt="" {...makeImg('sparkline', 'dark', index + 1)} />
-        </React.Fragment>
-      ))}
-      {[...Array(4)].map((_, index) => (
-        <React.Fragment key={index}>
-          <img alt="" {...makeImg('chart', 'light', index + 1)} />
-          <img alt="" {...makeImg('chart', 'dark', index + 1)} />
-        </React.Fragment>
-      ))}
-    </Box>
-  );
-}
 
 export default function XComponents() {
-  const [demo, setDemo] = React.useState(DEMOS[1]);
-  const icons = {
-    [DEMOS[0]]: <TableChartRounded fontSize="small" />,
-    [DEMOS[1]]: <DateRangeRounded fontSize="small" />,
-    [DEMOS[2]]: <BarChartRounded fontSize="small" />,
-    [DEMOS[3]]: <AccountTreeRounded fontSize="small" />,
-    [DEMOS[4]]: <ShowChartRounded fontSize="small" />,
-  };
+  const [componentIndex, setComponentIndex] = React.useState(0);
   return (
     <Section bg="gradient">
       <Grid container spacing={2}>
-        <Grid item md={6} sx={{ minWidth: 0 }}>
+        <Grid item md={6}>
           <SectionHeadline
-            overline="React component library"
+            overline="React advanced component libraries"
```

Comment:
Maybe rephrasing like this would make sense? 
```suggestion
            overline="Advanced React component libraries"
```

Reply:
Not sure "libraries" in plural fit here though - while each component has its own package, MUI X is an "Advanced React component library" (or a "collection" if you will)